invenio/base/config.py

```diff
@@ -601,7 +637,7 @@
 CFG_SITE_NAME = "Atlantis Institute of Fictive Science"
 CFG_SITE_RECORD = "record"
 SECRET_KEY = "change_me"
-CFG_SITE_SECURE_URL = "http://localhost:4000"
+CFG_SITE_SECURE_URL = "http://localhost:4040"
```

Comment:
You are changing default configuration here - please keep port 4000. This change should be discussed separately.